src/modalities/dataloader/dataloader.py

```diff
@@ -15,7 +15,7 @@ def __init__(
         batch_sampler: ResumableBatchSampler,
         dataset: Dataset[T_co],
         batch_size: Optional[int] = 1,
-        sampler: Union[Sampler, Iterable, None] = None,
+        sampler: Sampler | Iterable | None = None,
```

Comment:
```suggestion
        sampler: Optional[Sampler | Iterable] = None,
```
I think this is cleaner